### PR TITLE
YC-617: Set when a validation document is required

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://jira.liip.ch/browse/YC-617

### CODE REVIEW

File: geocity/tests/submissions/functional/test_submission_classify.py [hunk 1]
```diff
@@ -350,27 +351,118 @@ def _get_approval(self):
...
+    def test_classify_approve_submission_with_default_validation_text_shows_text(
+        self,
+    ):
+        default_validation_text = "Mon texte de validation par défaut"
+        form = factories.FormFactory(
+            default_validation_text=default_validation_text,
+        )
+        self.validation.submission.forms.set([form])
+        response = self._get_approval()
+        self.assertContains(response, default_validation_text)
```
Comment (monodo) [inline geocity/tests/submissions/functional/test_submission_classify.py:468]: You might want to add tests that ensures the classify step ("approve/reject") routes works correcty. For instance, reverser the route in the test and check that the submission statuts has changed ?

Comment (rbovard) [reply geocity/tests/submissions/functional/test_submission_classify.py:468]: It's already done here https://github.com/yverdon/geocity/blob/d4156d30c5506bc2622d3f3e32d4047833672624/geocity/tests/submissions/functional/test_submission_classify.py#L46-L64 and here https://github.com/yverdon/geocity/blob/d4156d30c5506bc2622d3f3e32d4047833672624/geocity/tests/submissions/functional/test_submission_classify.py#L97-L115

Comment (monodo) [reply geocity/tests/submissions/functional/test_submission_classify.py:468]: Good

**Comment by rbovard** [comment]
@monodo DRY refactor done, please review.

**Pull request merged**